src/eval_env.h

```diff
@@ -27,18 +26,18 @@ struct Rule;
 /// An interface for a scope for variable (e.g. "$foo") lookups.
 struct Env {
   virtual ~Env() {}
-  virtual string LookupVariable(const string& var) = 0;
+  virtual std::string LookupVariable(const std::string& var) = 0;
 };
 
-/// A tokenized string that contains variable references.
+/// A tokenized std::string that contains variable references.
```

Comment:
I think this was meant more as a general string, not a std::string in particular.